### DISCUSSION TITLE
A recap of Quarkus Tools for IntelliJ's latest improvements

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36370] [Quarkus Blog/Website]
# A recap of Quarkus Tools for IntelliJ's latest improvements

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/intellij-quarkus-recap/

**Comment by bentito** [comment]
Great, article! 

Just one nit: You might want to add a bit of disambiguation at the start of the article. IntelliJ seems to come bundled with a plugin called "Quarkus", at least this is how it's labeled for me. 

I happen to be on a freshly installed instance of IntelliJ, so for me, Quarkus plugin is installed by default and Quarkus Tools I'm adding back in.
